k8s/base/jitsi-meet-front-hpa.yml

```diff
@@ -0,0 +1,21 @@
+apiVersion: autoscaling/v2beta2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: jitsi-meet-front-hpa
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: jitsi-meet-front
+  minReplicas: 2
+  maxReplicas: 5
```

Comment:
could this be what was limiting our capacity during load tests?

Reply:
The main reason why the front HPA did not trigger is because we did not reach the CPU load required for a new pod to be scheduled.